dali/ft/filecopy.cpp

```diff
@@ -2491,6 +2491,37 @@ bool FileSprayer::needToCalcOutput()
     return !usePullOperation() || options->getPropBool(ANverify);
 }
 
+
+CriticalSection  maxConnectionsDefCrit;
```

Comment:
minor: should be a static.

trivial: double space before max.

Reply:
Changed